sdk/core/AzureCore/Source/Pipeline/PipelineClient.swift

```diff
@@ -59,6 +59,7 @@ open class PipelineClient {
     public var baseUrl: URL
     public var logger: ClientLogger
     public var commonOptions: AzureClientOptions
+    public var skipUrlEncoding: Bool
```

Comment:
Should we perhaps have an "internal" options object that is not exposed to customers, for things like this? I can imagine more flags being needed in the future and we don't want to explode the parameter list.

Reply:
Thanks for your review Brandon. After further discussion with Travis, we are changing our solution. Changes in this PR is not needed.